keyboards/bm60poker/info.json

```diff
@@ -1,74 +0,0 @@
-{
```

Comment:
Please restore this file.

Reply:
I'm getting conflicting information about this file @zvecr stated it didn't belong in this repo: https://github.com/qmk/qmk_firmware/pull/10987#discussion_r528228791